src/command/backup/backup.c

```diff
@@ -701,21 +705,82 @@ backupBuildIncr(
     FUNCTION_LOG_RETURN(BOOL, result);
 }
 
+/***********************************************************************************************************************************
+Get size of files to be copied in a manifest
+***********************************************************************************************************************************/
+static uint64_t
+backupManifestCopySize(Manifest *const manifest)
+{
+    FUNCTION_LOG_BEGIN(logLevelDebug);
+        FUNCTION_LOG_PARAM(MANIFEST, manifest);
+    FUNCTION_LOG_END();
+
+    ASSERT(manifest != NULL);
+
+    uint64_t result = 0;
+
+    for (unsigned int fileIdx = 0; fileIdx < manifestFileTotal(manifest); fileIdx++)
+    {
+        const ManifestFile file = manifestFile(manifest, fileIdx);
+
+        if (file.copy)
+            result += file.size;
+    }
+
+    FUNCTION_LOG_RETURN(UINT64, result);
+}
+
+/***********************************************************************************************************************************
+Get the last full backup time in order to set the limit for full/incr preliminary copy
```

Comment:
("naming things are hard", also it's pretty ingrained at this point, so take as a general grumble and not an actual action item.)

Reply:
I'm open to suggestions and totally happy to update the code.

Reply:
I dunno, these are more marketing-ish names, but
- accelerated backup? (kinda not true, since they can be longer)
- smart backup? (generic, non-descript)
- adaptive?

Don't really like any of them, so since this is still fairly user-invisible can probably punt on any great renaming until it gets more visibility.

Reply:
Thinking about "hybrid" multi steps backups, I initially thought about "two-phase backup" but didn't mention it since full/incr is quite intuitive after all once described 🤷

Reply:
Two-phase backup certainly sounds very professional but I think full/incr is more descriptive. Thoughts, @pgguru?

Reply:
Staged incremental? I think at this point I don't have super-strong feelings about any alternatives, so going with the original would be my default suggestion.

Reply:
OK. If we come up with a good idea before release we can always update it.